source/core/NumericString.cpp

```diff
@@ -257,83 +257,33 @@ void DefaultFormatCode(Int32 count, StaticTypeAndData arguments[], TempStackCStr
-    buffer->AppendUtf8Str(formatUnEscaped, newLength);
-    buffer->AppendCStr("'\n");
-    delete[] formatUnEscaped;
-}
-
-/**
- * Error reporting for when there is a format specifier provided but
- * not enough arguments provided with the Format.
- * */
-void ErrFormatMissingNumArgs(SubString* format, Int32 count, StaticTypeAndData arguments[], StringRef buffer, char formatCode) {
-    char formatSpecifierBuf[3];
-    snprintf(formatSpecifierBuf, 3, "%c", formatCode);
-    buffer->Resize1D(0);
-    buffer->AppendCStr("Error: Missing args provided for format string: '%");
-    buffer->AppendCStr(formatSpecifierBuf);
-    buffer->AppendCStr("' in '");
-    IntIndex length = format->Length()*2;
-    Utf8Char* formatUnEscaped = new Utf8Char[length];
-    IntIndex newLength = format->UnEscape(formatUnEscaped, length);
-    buffer->AppendUtf8Str(formatUnEscaped, newLength);
-    buffer->AppendCStr("'\n");
-    delete[] formatUnEscaped;
-}
-
-/**
- * Error reporting function for when the Format String is invalid
- * This will clear the current buffer string and return on what format
- * specifier it failed on.
- * */
-void ErrFormatInvalidFormatString(SubString* format, Int32 count, StaticTypeAndData arguments[], StringRef buffer, char formatCode) {
-    char formatSpecifierBuf[3];
-    snprintf(formatSpecifierBuf, 3, "%c", formatCode);
-    buffer->Resize1D(0);
-    buffer->AppendCStr("Error: Invalid format string provided: '%");
-    buffer->AppendCStr(formatSpecifierBuf);
-    buffer->AppendCStr("' in '");
-    IntIndex length = format->Length()*2;
-    Utf8Char* formatUnEscaped = new Utf8Char[length];
-    IntIndex newLength = format->UnEscape(formatUnEscaped, length);
-    buffer->AppendUtf8Str(formatUnEscaped, newLength);
-    buffer->AppendCStr("'\n");
-    delete[] formatUnEscaped;
-}
-
-/**
- * Error reporting function for when there is a type mismatch between 
- * arguments and the format specifiers.
- * This will clear the current buffer string and return on what format
- * specifier it failed on.
- * */
-void ErrMismatchedFormatSpecifier(SubString* format, Int32 count, StaticTypeAndData arguments[], StringRef buffer, char formatCode) {
-    char formatSpecifierBuf[3];
-    snprintf(formatSpecifierBuf, 3, "%c", formatCode);
-    buffer->Resize1D(0);
-    buffer->AppendCStr("Error: Invalid format specifier type mismatch: '%");
-    buffer->AppendCStr(formatSpecifierBuf);
-    buffer->AppendCStr("' in '");
-    IntIndex length = format->Length()*2;
-    Utf8Char* formatUnEscaped = new Utf8Char[length];
-    IntIndex newLength = format->UnEscape(formatUnEscaped, length);
-    buffer->AppendUtf8Str(formatUnEscaped, newLength);
-    buffer->AppendCStr("'\n");
-    delete[] formatUnEscaped;
+// Format Into String/Scan From String (StringFormat/StringCan) error codes
+enum { kFormatArgErr=1, kFormatTypeMismatch = 81, kFormatCodeUnknown, kFormatTooManyFormatSpecs, kFormatTooFewFormatSpecs, kFormatScanFailed };
```

Comment:
I'm curious as to how the error list was obtained and wonder if we didn't miss one.